invenio_search/ext.py

```diff
@@ -405,6 +405,20 @@ def _build(tree_or_filename, alias=None):
                     name=action["alias"],
                     ignore=ignore,
                 )
+    def update_mapping(self, index):
+        """Update mapping of the existing index."""
+        mapping_path = self.mappings[index]
+        final_index = build_alias_name(index)
+
+        index_ = self.client.indices.get(final_index)
+        full_index_name = list(index_.keys())[0]
```

Comment:
should we have `assert len(index.keys()) == 1` before? if we are going to assume we want the first there should be only one. Since we are using the full name it should always be ok, but IIRC mappings do not support field deletion so an error would be permanent.